src/components/characterPreview/CharacterScoringSummary.tsx

```diff
@@ -221,22 +220,28 @@ export const CharacterScoringSummary = (props: { simScoringResult: SimulationSco
-              <ScoringNumber label={t('CharacterPreview.ScoringColumn.PaddedStatLabels.SPD')} number={request.stats[Stats.SPD]} precision={2}/>
-              <ScoringNumber label={t('CharacterPreview.ScoringColumn.PaddedStatLabels.CR')} number={request.stats[Stats.CR]} precision={props.precision}/>
-              <ScoringNumber label={t('CharacterPreview.ScoringColumn.PaddedStatLabels.CD')} number={request.stats[Stats.CD]} precision={props.precision}/>
-              <ScoringNumber label={t('CharacterPreview.ScoringColumn.PaddedStatLabels.EHR')} number={request.stats[Stats.EHR]} precision={props.precision}/>
-              <ScoringNumber label={t('CharacterPreview.ScoringColumn.PaddedStatLabels.RES')} number={request.stats[Stats.RES]} precision={props.precision}/>
-              <ScoringNumber label={t('CharacterPreview.ScoringColumn.PaddedStatLabels.BE')} number={request.stats[Stats.BE]} precision={props.precision}/>
-            </Flex>
+            <ScoringDisplayColumn
+              style={{ width: 120 }} rowSpacing={defaultGap} columnSpacing={23}
+              values={[
+                { label: t(`common:ShortStats.${Stats.ATK_P}`) + ':', value: request.stats[Stats.ATK_P] || ' ', precision: props.precision },
+                { label: t(`common:ShortStats.${Stats.ATK}`) + ':', value: request.stats[Stats.ATK] || ' ', precision: props.precision },
+                { label: t(`common:ShortStats.${Stats.HP_P}`) + ':', value: request.stats[Stats.HP_P] || ' ', precision: props.precision },
+                { label: t(`common:ShortStats.${Stats.HP}`) + ':', value: request.stats[Stats.HP] || ' ', precision: props.precision },
+                { label: t(`common:ShortStats.${Stats.DEF_P}`) + ':', value: request.stats[Stats.DEF_P] || ' ', precision: props.precision },
+                { label: t(`common:ShortStats.${Stats.DEF}`) + ':', value: request.stats[Stats.DEF] || ' ', precision: props.precision },
+              ]}
+            />
+            <ScoringDisplayColumn
+              style={{ width: 100 }} rowSpacing={defaultGap} columnSpacing={31}
+              values={[
+                { label: t(`common:ShortStats.${Stats.SPD}`) + ':', value: request.stats[Stats.SPD] || ' ', precision: 2 },
+                { label: t(`common:ShortStats.${Stats.CR}`) + ':', value: request.stats[Stats.CR] || ' ', precision: props.precision },
+                { label: t(`common:ShortStats.${Stats.CD}`) + ':', value: request.stats[Stats.CD] || ' ', precision: props.precision },
+                { label: t(`common:ShortStats.${Stats.EHR}`) + ':', value: request.stats[Stats.EHR] || ' ', precision: props.precision },
+                { label: t(`common:ShortStats.${Stats.RES}`) + ':', value: request.stats[Stats.RES] || ' ', precision: props.precision },
+                { label: t(`common:ShortStats.${Stats.BE}`) + ':', value: request.stats[Stats.BE] || ' ', precision: props.precision },
+              ]}
+            />
```

Comment:
This feels like the wrong way to organize this unless there was a specific reason for removing ScoringNumber? I think it would be simpler to just change `<ScoringNumber>` to be a flex object with space-between. And make the number fixed width so it maintains decimal alignment

`[ [flex-start Label]                 [flex-end number] ]`